corehq/apps/userreports/templates/userreports/partials/report_filters.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% load userreports_tags %}
+
+<div class="hq-report-filters">
+  {% if report_filters %}
+    {% for filter in report_filters %}
+      <div class="mb-3">
+        <h4 class="form-label">{{ filter.label }}</h4>
+        {% with filter_context|get:filter.css_id as context_ %}
+          {{ context_.html|safe }}
+        {% endwith %}
+      </div>
+    {% endfor %}
+  {% else %}
+    <div class="alert alert-info">
+      {% trans "This report does not have any filters." %}
```

Comment:
Is this a pattern we use elsewhere? It seems nicer to me to just not show anything, rather than making the user read something.